### PR TITLE
antidote_app: Add file comment

### LINKED ISSUE #155: Add file comments for each source module
Each file contains only the copyright info. It would be good to have a 1-5 lines at the top of each file, describing the module.

### PR DESCRIPTION
**Pull request by palash25**
Related to: #155

### CODE REVIEW

File: src/antidote_app.erl [hunk 1]
```diff
@@ -25,6 +25,11 @@
 %% List of the contributors to the development of Antidote: see AUTHORS file.
 %% Description and complete License: see LICENSE file.
 %% -------------------------------------------------------------------
+
+%% @doc Starts the antidote supervisor process,
```
Comment (bieniusa) [inline src/antidote_app.erl]: I suggest adding as information that this module implements the application callbacks.

Comment (palash25) [reply src/antidote_app.erl]: would you like that to be added along with the current comment or do you want it to replace it? @bieniusa

Comment (bieniusa) [reply src/antidote_app.erl]: Adding :) Thanks!

**Comment by palash25** [comment]
@bieniusa amended please take a look

**Pull request merged**